content/en/docs/setup/additional-setup/gateway/high-level-canary.svg

```diff
@@ -0,0 +1,3 @@
1.88 L 320 283.63 L 323.5 281.88 Z" fill="#000000" stroke="#000000" stroke-miterlimit="10" pointer-events="all"/><rect x="235" y="150" width="170" height="60" rx="9" ry="9" fill="#fff2cc" stroke="#d6b656" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 168px; height: 1px; padding-top: 180px; margin-left: 236px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; "><font style="font-size: 16px">istio-ingress-canary Service</font></div></div></div></foreignObject><text x="320" y="184" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">istio-ingress-canary Service</text></switch></g><path d="M 220 60 L 220 105 L 100 105 L 100 143.63" fill="none" stroke="#000000" stroke-miterlimit="10" pointer-events="stroke"/><path d="M 100 148.88 L 96.5 141.88 L 100 143.63 L 103.5 141.88 Z" fill="#000000" stroke="#000000" stroke-miterlimit="10" pointer-events="all"/><path d="M 220 60 L 220 105 L 320 105 L 320 143.63" fill="none" stroke="#000000" stroke-miterlimit="10" pointer-events="stroke"/><path d="M 320 148.88 L 316.5 141.88 L 320 143.63 L 323.5 141.88 Z" fill="#000000" stroke="#000000" stroke-miterlimit="10" pointer-events="all"/><rect x="90" y="0" width="260" height="60" rx="9" ry="9" fill="#ffe6cc" stroke="#d79b00" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 258px; height: 1px; padding-top: 30px; margin-left: 91px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; ">External Load Balancer<br />OR<br />DNS configuration</div></div></div></foreignObject><text x="220" y="34" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">External Load Balancer...</text></switch></g></g><switch><g requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"/><a transform="translate(0,-5)" xlink:href="https://www.diagrams.net/doc/faq/svg-export-text-problems" target="_blank"><text text-anchor="middle" font-size="10px" x="50%" y="100%">Viewer does not support full SVG 1.1</text></a></switch></svg>
```

Comment:
Lint and netlify are failing. Something about this image is causing the image shortcode to get messed up here: https://github.com/istio/istio.io/blob/692f67753a7a4de42c37724c575ae836edadba8d/layouts/shortcodes/image.html#L91

Maybe it's a bug with negative viewBox values? `viewBox="-0.5 -0.5 421 431"`

Reply:
yeah I have to manually change to `0 0 ...`. Not sure why. I thought I did, let me fix it